.github/actions/linux-build/action.yml

```diff
@@ -91,6 +92,7 @@ runs:
         echo "docker pull ghcr.io/pytorch/ci-image:${tag/:/-}"
 
     - name: Pull docker image
+      if: inputs.build-environment != 'linux-s390x-binary-manywheel'
```

Comment:
Hmm, I wonder why you need this here, there should be a step to pull the Docker image `pytorch/manylinuxs390x-builder:cpu-s390x-main`, shoudn't it?

Reply:
Well, it tries to pull it from your amazon instance, and it's not there. For now I just baked this docker image into worker instance.

But I just noticed that it's possible to change where it downloads it from. Probably if this image is published somewhere, then this step can be actually done.